conans/client/cmd/uploader.py

```diff
@@ -164,10 +163,11 @@ def _prepare_package(self, pref, prev_bundle):
         prev_bundle["files"] = cache_files
 
     def _compress_package_files(self, layout, pref):
+        output = ConanOutput(scope=str(pref))
         download_pkg_folder = layout.download_package()
         package_tgz = os.path.join(download_pkg_folder, PACKAGE_TGZ_NAME)
         if is_dirty(package_tgz):
-            self._output.warning("%s: Removing %s, marked as dirty" % (str(pref), PACKAGE_TGZ_NAME))
+            output.warning("%s: Removing %s, marked as dirty" % (str(pref), PACKAGE_TGZ_NAME))
```

Comment:
```suggestion
            output.warning("Removing %s, marked as dirty" % PACKAGE_TGZ_NAME)
```

Otherwise it "scopes" it twice, no?

Reply:
Good catch